azerothcore/azerothcore-wotlk
# fix(Core/Player): Correct SPEC_MASK for addSpell()

## Changes Proposed:
- Change true to SPEC_MASK_ALL in addSpell() as it is not a bool =)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6527

## Tests Performed:

- Builds
- Tested in-game


## How to Test the Changes:


1. Create a new char: See you can talk and have spells
2. Get Dual Spec, see that you can talk and have spells even when you change spec
3. Change back and you can still talk and have spells


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
